au4/src/au3wrap/internal/domaccessor.h

```diff
@@ -19,6 +19,7 @@ class DomAccessor
     static WaveTrack* findWaveTrack(AudacityProject& prj, const TrackId& au3trackId);
     static std::shared_ptr<WaveClip> findWaveClip(WaveTrack* track, uint64_t au3ClipId);
     static std::shared_ptr<WaveClip> findWaveClip(AudacityProject& prj, const TrackId& au3trackId, size_t index);
+    static std::shared_ptr<WaveClip> findWaveClipAtGivenTime(AudacityProject &prj, const trackedit::TrackId &trackId, trackedit::secs_t time);
```

Comment:
I think we can name it just `findWaveClip`
"AtGivenTime" can be understood from the input params

Reply:
fixed